querybook/webapp/components/DataTableViewSourceQuery/DataTableViewSourceQuery.tsx

```diff
@@ -80,6 +81,21 @@ export const DataTableViewSourceQuery: React.FunctionComponent<IProps> = ({
         );
     });
 
+    const customProperties = table.custom_properties ?? {};
+    const workflowValue = customProperties['workflow']?.toString() ?? null;
+    const workflowDOM = workflowValue ? (
+        <div className="DataTableViewSourceQuery-workflow">
+            <div className="data-job-source-query-top">
```

Comment:
this class `data-job-source-query-top` is from component `DataJobMetadataInfo`, looks weird to be here. better to define its own.